consensus/safety-rules/src/tests/suite.rs

```diff
@@ -664,7 +664,10 @@ fn test_validator_not_in_set(safety_rules: &Callback) {
     proof
         .ledger_info_with_sigs
         .push(a2.block().quorum_cert().ledger_info().clone());
-    safety_rules.initialize(&proof).unwrap();
+    match safety_rules.initialize(&proof) {
```

Comment:
you can use matches! macro

Reply:
Ooh, nice, thanks! Didn't know we had matches on stable 😄